src/components/security_manager/src/ssl_context_impl.cc

```diff
@@ -596,7 +596,7 @@ void CryptoManagerImpl::SSLContextImpl::ResetConnection() {
     SSL_shutdown(connection_);
   }
   LOG4CXX_DEBUG(logger_, "SSL connection recreation");
-  SSL_CTX* ssl_context = connection_->ctx;
+  SSL_CTX* ssl_context = SSL_get_SSL_CTX(connection_);
```

Comment:
@alexkutsan is this change necessary?

Reply:
Yep. nerw version of libssl is not compatible with previous code